.github/workflows/build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           docker image pull "$DOCKER_IMAGE"
       - name: Build manylinux wheels
         env:
-          PYTHON_TAGS: 'cp35-cp35m cp36-cp36m cp37-cp37m'
+          PYTHON_TAGS: 'cp35-cp35m cp36-cp36m cp37-cp37m cp38-cp38'
```

Comment:
should that be `cp38-cp38m`?

Reply:
The 'm' is gone in 3.8 (https://github.com/pypa/manylinux/blob/master/README.rst#manylinux2014): "Note that starting with CPython 3.8, default sys.abiflags became an empty string: the m flag for pymalloc became useless (builds with and without pymalloc are ABI compatible) and so has been removed. (e.g. /opt/python/cp38-cp38)"